release_notes/44-release-notes.adoc

```diff
@@ -563,6 +565,42 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
 +
 This update ensures that the linked collection is retained, avoiding the error message.
 
+[id="resolved-in-version-441_{context}"]
+=== Resolved in version 4.4.1
+
+*Release date*: 22 April 2024
+
+* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.
+
+[id="resolved-in-version-442_{context}"]
+=== Resolved in version 4.4.2
+
+*Release date*: 13 May 2024
+
+This release provides the following bug fixes:
+
+* Before this update, Collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
+* This release updates the Scanner baseline vulnerability data to address changes made to the Red{nbsp}Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.
+* This release fixes a crash and rendering error in the network graph that occurs when Central is running an {product-title-short} release of 4.3.6 or earlier and Sensor is running an {product-title-short} release of 4.4.0 or later.
+
+This release provides the following change:
+
+* The default telemetry endpoint is now set to a Red{nbsp}Hat proxy.
+
+This releases updates the following items to patch vulnerabilities:
+
+* Go has been updated to release 1.20.12.
+* The `golang.org/x/net` module has been updated from release v0.22.0 to v0.23.0.
+
```

Comment:
**Note:** This is the only modified part. Rest of the content is already reviewed and published in `rhacs-docs-4.4` branch but missing from the `rhacs-docs-main` branch.